app/gui/view/component-browser/component-group/src/lib.rs

```diff
@@ -462,6 +468,13 @@ impl Model {
         self.header.add_to_scene_layer(&layers.header_text);
     }
 
+    /// Whether the `point` (object-space coordinates) is inside the component group shape.
```

Comment:
Minor thing: Comments should be written in active form. Use "Test whether ..." instead. Pls fix it here and in other comments.